server/api/eventinfo/eventinfo.controller.js

```diff
@@ -76,9 +107,13 @@ exports.show = function(req, res) {
 
 // Creates a new EventInfo in the DB
 exports.create = function(req, res) {
-  EventInfo.createAsync(req.body)
+  saveEventInfoImage(req.files, req.body)((imagePaths)=>{
+    var evnt = req.body;
+    evnt.imageURLs = imagePaths;
+    EventInfo.createAsync(evnt)
     .then(responseWithResult(res, 201))
     .catch(handleError(res));
+  });
```

Comment:
pass the callback as a parameter to the function, the callback shouldn't be returned